clients/ts/signalr/package.json

```diff
@@ -36,6 +36,12 @@
     "src/**/*"
   ],
   "devDependencies": {
-    "es6-promise": "^4.2.2"
+    "es6-promise": "^4.2.2",
+    "@types/websocket": "^0.0.40",
+    "@types/eventsource": "^1.0.2"
+  },
+  "dependencies": {
```

Comment:
@BrennanConroy do these become something that we re-ship embedded in our own JS file? Or just used at build time, and customers would get them via NPM?

Reply:
These are runtime dependencies acquired by the **user** when they install our package. We do not embed them in any of our JavaScript files.

Reply:
OK cool then I'll wait for @anurse 's OSS integration thingy to add these to our OSS list. No immediate action required here.

Reply:
@Eilon ?

Reply:
We need a 3PN in any case where we redistribute 3rd party code. This can be either because we've checked in 3rd party code into a GitHub repo we own (because we own the repo, we are the ones redist'ing the code), or this can be because we produce a package (e.g. NUPKG, NPM, etc.) that includes 3rd party code (e.g. our template NUPKGs contain jQuery).

If the user acquires the 3rd party code via another mechanism (e.g. directly from NuGet.org via a dependency), then 3PN is not needed.

Reply:
Cool, thanks for the clarification! No 3pn needed here then, we don't embed anything new.